src/lib/y2storage/proposal/autoinst_nfs_planner.rb

```diff
@@ -0,0 +1,212 @@
+# encoding: utf-8
+
+# Copyright (c) [2019] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "y2storage/proposal/autoinst_drive_planner"
+
+module Y2Storage
+  module Proposal
+    # This class converts an AutoYaST specification into a Planned::Nfs in order
+    # to set up a NFS filesystem.
+    class AutoinstNfsPlanner < AutoinstDrivePlanner
+      # Returns an array of planned NFS filesystems according to an AutoYaST specification
+      #
+      # @param drive [AutoinstProfile::DriveSection] drive section describing NFS filesystems
+      # @return [Array<Planned::Nfs>] Planned NFS filesystems
+      def planned_devices(drive)
+        # TODO: planned device with new format
+
+        planned_devices_old_format(drive)
+      end
+
+    private
+
+      NEW_FORMAT_MANDATORY_VALUES = { drive: [:device], partition: [:mount] }.freeze
```

Comment:
can you document what this hash means? If someone will modify it in future, so he knows. ( like what is key and what is value ).